extensions/amp-consent/customizing-extension-behaviors-on-consent.md

```diff
@@ -70,3 +70,11 @@ Ways to get the consent string for ad/analytics vendors coming soon.
 In addition to the consent state and consent string, AMP extensions can also use the
 `getConsentPolicySharedData` API to receive additional consent related information about
 the user from the page owner. See [this](https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#response) for details about the `shareData`.
+
+### On Consent Metadata
+
+In addition to the consent state and consent string, AMP extensions can also use the
+`getConsentMetadata` API to receive additional consent metadata information about
+the user from the page owner (`additionalConsent`, `gdprApplies`, `consentStringType`) via the `checkConsentHref` endpoint or from the CMP. AMP will pass this metadata to vendors without modification.
```

Comment:
The `page owner` is unclear here, it's CMP who provide the extra data, not pub. What about

In addition to the consent state and consent string, AMP extensions can also use the `getConsentMetaData` to receive additional consent information AMP receives from the `checkConsentHref` or from the CMP. 

Same to the consent string, AMP will always pass the local stored consent metadata object if there's one. Update will only be reflected the next page load.

Reply:
I think it's useful to document the `consentMetadata` object here again because of three enum values. `CONSENT_STRING_TYPE.TCF_V1` `CONSENT_STRING_TYPE.TCF_V2` and `CONSENT_STRING_TYPE.US_PRIVACY_STRING`. (Vendors are not expected to use 1, 2, 3 like what we write in the CMP facing doc)